openedx/features/course_duration_limits/config.py

```diff
@@ -11,9 +11,3 @@
     flag_name=u'debug',
     flag_undefined_default=False
 )
-
-CONTENT_TYPE_GATING_STUDIO_UI_FLAG = WaffleFlag(
```

Comment:
could you clarify why this is deleted?

Reply:
It was replaced by the `override_studio_ui` flag in `ContentTypeGatingConfig`.